Formula/arcs.rb

```diff
@@ -20,6 +20,7 @@ class Arcs < Formula
 
   depends_on "boost" => :build
   depends_on "google-sparsehash" => :build
+  depends_on "links-scaffolder"
```

Comment:
```suggestion
```
I'm inclined to omit `links-scaffolder` as a dependency, unless it's required to build ARCS.

Reply:
My reasoning here is that the `arcs-make` Makefile, which most users use as the driver for running ARCS, requires LINKS

Reply:
I'm inclined to omit soft dependencies like this, although I agree that this is a grey-area case.